app/src/main/java/com/everest/moviedb/CurrentMovieFragment.kt

```diff
@@ -0,0 +1,70 @@
+package com.everest.moviedb
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.everest.moviedb.databinding.CurrentMovieLayoutBinding
+import com.everest.moviedb.ui.MovieDetail
+import com.everest.moviedb.utils.MOVIE_DETAILS
+
+class CurrentMovieFragment : Fragment(R.layout.current_movie_layout) {
+
+    private lateinit var binding: CurrentMovieLayoutBinding
+    private lateinit var viewModel: MovieViewModel
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        binding = CurrentMovieLayoutBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        val movieRepository =
```

Comment:
Just a suggestion check Hilt ( Dependency injection library), to apply dependency injecting easily 
- Not a blocker for this PR